src/scripts/install_podman.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -ex
+
+if [ "$ENGINE" == "podman" ]; then
+    . /etc/os-release
+    echo "deb https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/testing/xUbuntu_${VERSION_ID}/ /" | sudo tee /etc/apt/sources.list.d/devel:kubic:libcontainers:testing.list
+    curl -L https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/testing/xUbuntu_${VERSION_ID}/Release.key | sudo apt-key add -
+    sudo apt-get update -qq
+    sudo apt-get -qq -y install slirp4netns podman
```

Comment:
Oh, it's needed when installing podman in the travis environment. Both debian and alpine try to find it but don't see it hence trigger an error. Installing that into the environment fixes that.

Reply:
if this package is needed by the `podman` package, then that will be done automatically by `apt`. For example:

```
root@70c7503fe555:/# apt install podman
Reading package lists... Done
Building dependency tree
Reading state information... Done
The following additional packages will be installed:
  catatonit conmon containernetworking-plugins containers-common containers-golang containers-image
  crun dmsetup iptables libdevmapper1.02.1 libglib2.0-0 libglib2.0-data libgpgme11 libicu60 libip4tc0
  libip6tc0 libiptc0 libmnl0 libnetfilter-conntrack3 libnfnetlink0 libxml2 libxtables12 libyajl2
  multiarch-support podman-plugins runc shared-mime-info slirp4netns uidmap varlink xdg-user-dirs
Suggested packages:
  kmod
The following NEW packages will be installed:
  catatonit conmon containernetworking-plugins containers-common containers-golang containers-image
  crun dmsetup iptables libdevmapper1.02.1 libglib2.0-0 libglib2.0-data libgpgme11 libicu60 libip4tc0
  libip6tc0 libiptc0 libmnl0 libnetfilter-conntrack3 libnfnetlink0 libxml2 libxtables12 libyajl2
  multiarch-support podman podman-plugins runc shared-mime-info slirp4netns uidmap varlink
  xdg-user-dirs
0 upgraded, 32 newly installed, 0 to remove and 6 not upgraded.
Need to get 37.3 MB of archives.
After this operation, 181 MB of additional disk space will be used.
```

see `slirp4netns` above

Reply:
Is this the same as the Travis environment? Let me retry that.

Reply:
I tried removing `slirp4netns` in my [local](https://travis-ci.com/github/edeediong/popper/jobs/358297860) Travis and it failed. It actually requires `slirp4netns`. The above snippet is it from a Travis environment, a bare metal machine or a virtual machine?

Reply:
Same with [popper](https://travis-ci.com/github/getpopper/popper/jobs/358297891) travis environment. It seems to me it doesn't get installed into the VM travis provides.

Reply:
I tested in an ubuntu:18.04 docker image, and copy-pasted the commands, but excluded `slirp4netns`. Weird... we can leave it as you have it then, it might be something related to the travis images. Sorry for the noise

Reply:
No it's fine. That's what I was trying to say earlier, all environments didn't throw an error except Travis. So hopefully no one will encounter this issue when using on their local.